dolphinscheduler-standalone-server/src/main/resources/application.yaml

```
@@ -191,7 +191,7 @@ master:
   server-load-protection:
     enabled: true
     # Master max cpu usage, when the master's cpu usage is smaller then this value, master server can execute workflow.
-    max-cpu-usage-percentage-thresholds: 0.9
+    max-cpu-usage-percentage-thresholds: 1.5
```

```suggestion
    max-cpu-usage-percentage-thresholds: 1.0
```
The max value of this parameter shouldld be 1.0.